app/views/admin/institutions/_form.html.erb

```diff
@@ -20,9 +20,12 @@
     <%= form.label :introductory_text %>
     <%= form.text_area :introductory_text, class: 'form-control' %>
   </div>
+  <div class="form-group">
+    <%= form.label 'Transcript consensus number', for: 'institution_max_line_edits' %>
+    <%= form.text_field :max_line_edits, class: 'form-control' %>
```

Comment:
Just wondering if these match up? The `for` in the label should match the `id` in the text field

Reply:
hey @Rhiana , I checked and it's working as is :)

![2018-07-17 09 47 28](https://user-images.githubusercontent.com/98407/42789069-80b9f20e-89a6-11e8-9cf2-4e7d2cefcfd9.gif)